directory-listing-template.html

```diff
@@ -1,52 +1,48 @@
 <!DOCTYPE html>
 
-<html i18n-values="dir:textdirection;lang:language">
+<html dir="ltr" lang="en">
```

Comment:
oh nice. i copied this from chrome's builtin file:/// protocol viewer, i guess i didn't notice this

Reply:
I copied this from chromes built in thing too, I just used an updated version